pylintrc

```diff
@@ -0,0 +1,51 @@
+[MESSAGES CONTROL]
+disable=attribute-defined-outside-init,
```

Comment:
That's a minimal set of disabled checks to make pylint happy with the current codebase. I'm going to enable some of these checks one by one in subsequent PRs.